serialization/dotnet/json/src/JsonParseNode.cs

```diff
@@ -208,7 +216,7 @@ public IEnumerable<T> GetCollectionOfPrimitiveValues<T>()
             if(item.AdditionalData == null)
                 item.AdditionalData = new Dictionary<string, object>();
 
-            foreach(var fieldValue in _jsonNode.EnumerateObject().Where(x => x.Value.ValueKind != JsonValueKind.Null))
+            foreach(var fieldValue in _jsonNode.EnumerateObject())
```

Comment:
Null values are not added to additional data if this filter is applied here and an "unknown" property is returned from the api.

Reply:
thanks for the additional context